Pod/Classes/Main/StylingKit.m

```diff
@@ -110,14 +110,25 @@ - (void)startStyling
 - (STKTheme*)registerThemeNamed:(NSString*)themeName
                        inBundle:(NSBundle*)bundle
 {
+    return [self registerThemeNamed:themeName stylesheetFileName:nil inBundle:bundle];
+}
+
+- (STKTheme*)registerThemeNamed:(NSString*)themeName
+             stylesheetFileName:(NSString*)stylesheetFileName
+                       inBundle:(NSBundle*)bundle
+{
+    
+    
+    
     if (self.themes[themeName])
     {
         DDLogWarn(@"Theme with name %@ already registered. %@", themeName, self.themes[themeName]);
     }
     STKTheme* theme = [STKTheme themeWithName:themeName
+                           stylesheetFileName:(NSString*)stylesheetFileName
                                        bundle:bundle];
     self.themes[themeName] = theme;
-
+    
```

Comment:
please remove whitespaces at empty line (there is setting in Xcode to do this for you automatically)